src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -466,19 +466,69 @@ public enum ViewAccessibilityMode
             Custom,
         }
 
-        private static IntPtr NewWithAccessibilityMode(ViewAccessibilityMode accessibilityMode)
+        /// <summary>
+        /// ResizePolicy mode for controlling View's Relayout implementation.
+        /// It will be used when we can ensure that current view
+        /// will not use ResizePolicy and Relayout signal.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum ViewResizePolicyMode
+        {
+            /// <summary>
+            /// Default implementation. Will consider ResizePolicy and Relayout implementations
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Default,
+            /// <summary>
+            /// Ignore ResizePolicy and relative functions.
+            /// </summary>
+            /// <remarks>
+            /// It will be useful when we can assume that
+            /// - This view always be exist under Layout
+            /// - ResizePolicy don't need
+            /// - Relayout event not required
+            /// - Internal visuals, or VisualObjects don't use FittingMode.
+            /// - Text are not be used
```

Comment:
"are not be used" is slightly weird.

Text cannot use "Ignore", it ignores this property.
Two ignore is still not good. We need to ask it to ai.